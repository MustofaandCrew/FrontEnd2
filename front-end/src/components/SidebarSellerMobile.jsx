import React from "react";
import imageProfil from "../assets/images/user_pc.png";
import Logo from '../assets/images/logo.png';
import "../assets/css/SidebarSellerMobile.css";

export default function SideSellerMobile() {
    return (
        <nav className="navbar bg-light fixed-top">
            <div className="container-fluid">
                <button className="navbar-toggler" type="button" data-bs-toggle="offcanvas" data-bs-target="#offcanvasNavbar" aria-controls="offcanvasNavbar">
                    <span className="navbar-toggler-icon"></span>
                </button>
                <div className="offcanvas offcanvas-start" tabindex="-1" id="offcanvasNavbar" aria-labelledby="offcanvasNavbarLabel">
                    <div className="offcanvas-header">
                        <h5 className="offcanvas-title" id="offcanvasNavbarLabel"><a href="/" ><img src={Logo} className="logo-sidebar-mobile" /></a>SecondHand</h5>
                        <button type="button" className="btn-close" data-bs-dismiss="offcanvas" aria-label="Close"></button>
                    </div>
                    <div className="offcanvas-body">
                        <ul className="navbar-nav justify-content-end flex-grow-1 pe-3">
                            <li className="nav-item item-mobile text">
                                <a className="nav-link active" aria-current="page" href="/transaksibuyer">Dashboard</a>
                            </li>
                            <li className="nav-item item-mobile text">
                                <a className="nav-link active" href="/accountbuyer">My Product</a>
                            </li>
                            <li className="nav-item item-mobile text">
                                <a className="nav-link active" href="/accountbuyer">My Account</a>
                            </li>
                        </ul>
                    </div>
                </div>
                <a className="navbar-brand" href="#"><img src={imageProfil} alt="profil" className="foto-profil" />Hi, Angga</a>
            </div>
        </nav>
    )
}
