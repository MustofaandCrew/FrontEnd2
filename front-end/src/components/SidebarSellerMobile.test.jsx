import React from "react";
import renderer from "react-test-renderer";
import SidebarSellerMobile from "../components/SidebarSellerMobile";


describe("Test suits for SidebarSellerMobile", () => {
    it("should match with snapshot", () => {
        const tree = renderer.create(<SidebarSellerMobile />).toJSON();
    });
});