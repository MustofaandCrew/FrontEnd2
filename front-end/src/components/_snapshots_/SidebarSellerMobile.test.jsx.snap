// Jest Snapshot v1, https://goo.gl/fbAQLP
import SidebarSellerMobile from "../components/SidebarSellerMobile";

exports["SidebarSellerMobile"] = {
    test: function test() {
        var tree = renderer.create(<SidebarSellerMobile />).toJSON();
        expect(tree).toMatchSnapshot();
    }
};